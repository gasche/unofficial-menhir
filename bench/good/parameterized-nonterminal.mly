%token A
%start<int> a
%%

a: b(A) {}

b(X): c(X) d {}

d: A {}
c(X): X {}
