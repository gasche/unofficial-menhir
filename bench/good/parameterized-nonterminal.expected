%start a
%token A

%type <int> a
%%

a:
| b_A_
    {}

b_A_:
| c_A_ d
    {}

d:
| A
    {}

c_A_:
| A
    {}

%%



